packages/span-transformations/lib/jaeger.ts

```diff
@@ -0,0 +1,51 @@
+import { SpanAttributes, SpanKind } from '@opentelemetry/api';
+import { timeInputToHrTime } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
+
+const getJaegerValueForTag = (jaegerTagKey, tags) => tags.filter(({ key }) => jaegerTagKey === key)?.[0]?.value;
+const convertJaegerTagsToAttributes = (tags): SpanAttributes => {
+    const spanAttributes: SpanAttributes = {};
+    tags.forEach(({ key, value }) => {
+        spanAttributes[key] = value;
+    });
+    return spanAttributes;
+};
+
+const getOtelKindFromJaegerKind = (jaegerKind: string) => {
+    switch (jaegerKind) {
+        case 'client':
+            return SpanKind.CLIENT;
+        case 'producer':
+            return SpanKind.PRODUCER;
+        case 'server':
+            return SpanKind.SERVER;
+        case 'consumer':
+            return SpanKind.CONSUMER;
+        default:
+            return SpanKind.INTERNAL;
+    }
+};
+
+export const convertJaegerSpanToOtel = (jaegerSpan): ReadableSpan => {
+    const durationMillis = jaegerSpan.duration / 1000;
+    const startDateMillis = jaegerSpan.startTime / 1000;
+    const endDateMillis = timeInputToHrTime(new Date(startDateMillis + durationMillis));
+    return {
+        name: jaegerSpan.operationName,
+        kind: getOtelKindFromJaegerKind(getJaegerValueForTag('span.kind', jaegerSpan.tags)),
+        attributes: convertJaegerTagsToAttributes(jaegerSpan.tags),
+        duration: timeInputToHrTime(durationMillis),
+        startTime: timeInputToHrTime(startDateMillis),
+        endTime: endDateMillis,
+        links: [],
```

Comment:
how are links converted to jaeger?
If it's not supported lets document it

Reply:
It is supposed to by converted by reference. Personally I feel it's good enough without links & events for now (due to lack of time too).

I documented in the readme the exact fields we currently don't support.

Reply:
Cool,
Is it difficult to add support for those missing fields?

Reply:
It's not trivial. I don't have data on how logs look like on jaeger, it is always null as far as i can see so hard to convert from something you don't know how looks.

As for links, not too hard but if i want it to be complete i need to dive into it - which attributes needed, which not etc and i don't know if we want to go there now. @habmic would love your thoughts here too. 

(as long as it is documented we don't support i am fine with it as is for now.)

Reply:
I think we can document it is not supported